book/src/csi-node-info-object.md

```diff
@@ -2,11 +2,11 @@
 
 ## Status
 
-Alpha
+Beta
```

Comment:
Add the following line underneath:

The `CSINodeInfo` object was introduced as alpha in Kubernetes v1.12. It was promoted to beta in Kubernetes 1.13.